src/pages/office/__tests__/office.spec.ts

```diff
@@ -42,6 +75,27 @@ describe('OfficePage', () => {
     });
   });
 
+  describe('DOM', () => {
+    it('should hide ETA faults container if there are none', () => {
+      fixture.detectChanges();
+      expect(fixture.debugElement.query(By.css('#ETA'))).toBeNull();
+    });
+    it('should display ETA faults container if there are any', () => {
```

Comment:
Is it worth testing the Physical ETA here or are we sure it works for both based on tests elsewhere